config/config.exs

```diff
@@ -5,7 +5,13 @@ config :commuter_rail_boarding,
   stop_ids: %{
     "South Station" => "NEC-2287",
     "North Station" => "BNT-0000",
-    "Back Bay" => "NEC-2276",
+    "Back Bay" => %{
+      "1" => "NEC-2276",
+      "2" => "NEC-2276",
+      "3" => "NEC-2276",
+      "5" => "WML-0012",
+      "7" => "WML-0012"
```

Comment:
question: should these be "WML-0012-05" and "WML-0012-07"? or are those used somewhere else?

Reply:
For boarding statuses with a track assignment, the stop ID in the TripUpdate is determined by taking this value and appending `-` plus the track number, zero-padded to two digits ([code here](https://github.com/mbta/commuter_rail_boarding/blob/b3a02cb66a7368f8e09e22c3853810125495b40c/apps/commuter_rail_boarding/lib/trip_updates.ex#L118)). This seems a bit suspect to me, since we are relying on stop IDs having a specific format rather than being opaque, but taking the current system as a given I'm pretty sure this is using it correctly.